tests/next_tests/integration_tests/feature_tests/ffront_tests/test_gt4py_builtins.py

```diff
@@ -95,16 +95,9 @@ def reduction_ek_field(
     return neighbor_sum(edge_f(V2E), axis=V2EDim)
 
 
-@gtx.field_operator
```

Comment:
@havogt We could also do an xfail here if you prefer. It's technically a little challenging, but would be possible. What do you prefer?